Common.props

```diff
@@ -1,11 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), 'Global.props'))\Global.props" />
-  <Import Project=".\GlobalStaticVersion.props" />
```

Comment:
this change makes this repository inconsistent with the https://github.com/Microsoft/ApplicationInsights-dotnet Do you think we need to change that one too?